File: java/src/test/java/org/taymyr/lagom/javadsl/broker/SimpleTopicProducerTestModule.java
package org.taymyr.lagom.javadsl.broker;

import com.google.inject.AbstractModule;
import com.lightbend.lagom.javadsl.server.ServiceGuiceSupport;

public class SimpleTopicProducerTestModule extends AbstractModule implements ServiceGuiceSupport {

    @Override
    protected void configure() {
        bindService(TestTopicPublisher.class, TestTopicPublisherImpl.class);
    }
}

File: java/src/test/kotlin/org/taymyr/lagom/javadsl/broker/SimpleTopicProducerTestModule.kt
package org.taymyr.lagom.javadsl.broker

import com.google.inject.AbstractModule
import com.lightbend.lagom.javadsl.server.ServiceGuiceSupport

class SimpleTopicProducerTestModule : AbstractModule(), ServiceGuiceSupport {
    override fun configure() {
        bindService(TestTopicPublisher::class.java, TestTopicPublisherImpl::class.java)
    }
}